src/Nordea/Components/FileDownload.elm

```diff
@@ -0,0 +1,143 @@
+module Nordea.Components.FileDownload exposing (..)
+
+import Css
+    exposing
+        ( alignItems
+        , auto
+        , border3
+        , borderRadius
+        , center
+        , cursor
+        , displayFlex
+        , flexBasis
+        , flexShrink
+        , lineHeight
+        , marginBottom
+        , marginLeft
+        , marginRight
+        , middle
+        , num
+        , padding
+        , pct
+        , pointer
+        , pseudoClass
+        , rem
+        , solid
+        , textDecoration
+        , underline
+        , verticalAlign
+        , width
+        )
+import Css.Global as Css exposing (children, everything)
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Html.Styled.Events exposing (onClick)
+import Nordea.Components.Spinner as Spinner
+import Nordea.Components.Text as Text
+import Nordea.Html as Layout
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icon
+import Nordea.Themes as Themes
+
+
+type alias Translation =
+    { no : String, se : String, dk : String, en : String }
+
+
+type alias Translate =
+    Translation -> String
+
+
+type FileDownload msg
+    = FileDownload (Config msg)
+
+
+type alias Config msg =
+    { onDownload : msg
+    , label : String
+    , type_ : Maybe String
+    , id : String
```

Comment:
Is this used?
```suggestion
```

Reply:
It doesnt use anywhere. I will remove it 😊